docker/bookkeeper/Dockerfile

```diff
@@ -40,7 +54,4 @@ RUN chmod +x -R /opt/bookkeeper/scripts/
 ENTRYPOINT [ "/bin/bash", "/opt/bookkeeper/scripts/pravega_entrypoint.sh" ]
 CMD ["bookie"]
 
-# BookKeeper healthcheck was broken in 4.7.3 and was not fixed until 4.9.0
-# This overrides the healthcheck to the default BK healthcheck method
-# https://github.com/apache/bookkeeper/issues/1687
-HEALTHCHECK --interval=10s --timeout=60s CMD /opt/bookkeeper/bin/bookkeeper shell bookiesanity
+HEALTHCHECK --interval=10s --timeout=60s CMD /bin/bash /opt/bookkeeper/scripts/healthcheck.sh
```

Comment:
According to the previous comment here, the health check was not fixed until `4.9.0`. But this PR is now targeting 4.8.2. Are you sure that this works in its current form?

Reply:
@RaulGracia yes you are right. This was an oversight. I've fixed it now.